content/registry/tools-python-google-cloud.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Google Cloud Tools
+registryType: tools
```

Comment:
There might be a better category name for this than "tools". The package has a trace propagator and a resource detector. Thoughts?

Reply:
'extensions', maybe?

Reply:
I noticed the registry page says (https://opentelemetry.io/registry/)
>The OpenTelemetry Registry allows you to search for instrumentation libraries, tracer implementations, utilities, and other useful projects in the OpenTelemetry ecosystem.

Maybe "utilities" then?

Reply:
yeah, let's go with that, it's probably the most accurate for now.